sdks/python/apache_beam/utils/proto_utils.py

```diff
@@ -123,8 +126,25 @@ def pack_Struct(**kwargs):
 def from_micros(cls, micros):
   # type: (Type[TimeMessageT], int) -> TimeMessageT
   result = cls()
-  result.FromMicroseconds(micros)
-  return result
+  if isinstance(result, duration_pb2.Duration):
+    result.FromMicroseconds(micros)
+    return result
+  elif isinstance(result, timestamp_pb2.Timestamp):
+    result.seconds = micros // _MICROS_TO_SECONDS
```

Comment:
can we add some comments here about why we do these conversions?

Reply:
+1. add brief comment why we don't use   timestamp_pb2.Timestamp.FromMicroseconds()
  and a reference an issue that has details.